src/coreclr/jit/block.h

```diff
@@ -1419,6 +1419,8 @@ struct BasicBlock : private LIR::Range
         int      bbHistogramSchemaIndex; // schema index for histogram instrumentation
     };
 
+    int bbValueSchemaIndex; // schema index for value instrumentation
```

Comment:
Count and HandleHistogram each have their own index fields, Value probing used to use Handle's one and it could lead to asserts. This field doesn't increase BasicBlock's layout (it had paddings) - still same `272` bytes on 64bit.